packages/application-config/package.json

```diff
@@ -31,7 +31,9 @@
     "README.md"
   ],
   "scripts": {
-    "build:schema": "json2ts schema.json src/schema.ts --style.singleQuote --bannerComment '/* eslint-disable prettier/prettier */\n// This file was automatically generated by json-schema-to-typescript.\n// DO NOT MODIFY IT BY HAND. Instead, modify the source schema.json file.'"
+    "build:customApplicationSchema": "json2ts custom-application.schema.json src/custom-application.schema.ts --style.singleQuote --bannerComment '/* eslint-disable prettier/prettier */\n// This file was automatically generated by json-schema-to-typescript.\n// DO NOT MODIFY IT BY HAND. Instead, modify the source custom-application.schema.json file and run the build:schemas npm script.'",
+    "build:customViewSchema": "json2ts custom-view.schema.json src/custom-view.schema.ts --style.singleQuote --bannerComment '/* eslint-disable prettier/prettier */\n// This file was automatically generated by json-schema-to-typescript.\n// DO NOT MODIFY IT BY HAND. Instead, modify the source custom-view.schema.json file and run the build:schemas npm script.'",
+    "build:schemas": "pnpm build:customApplicationSchema && pnpm build:customViewSchema"
```

Comment:
We now have to manage two schemas for configuration files (Custom Applications and Custom Views).